js/pro/cex.js

```diff
@@ -0,0 +1,1216 @@
+        //       }
+        //
+        const isTransaction = this.safeValue (order, 'd') !== undefined;
+        const remainsPrecision = this.safeString (order, 'remains');
+        let remaining = undefined;
+        if (remainsPrecision !== undefined) {
+            remaining = this.currencyFromPrecision (market['base'], remainsPrecision);
+        }
+        const amount = this.safeString (order, 'amount');
+        if (!isTransaction) {
+            this.currencyFromPrecision (market['base'], amount);
+        }
+        let baseId = this.safeString (order, 'symbol');
+        let quoteId = this.safeString (order, 'symbol2');
+        const pair = this.safeValue (order, 'pair');
+        if (pair !== undefined) {
+            baseId = this.safeString (order, 'symbol1');
+            quoteId = this.safeString (order, 'symbol2');
+        }
+        const base = this.safeCurrencyCode (baseId);
+        const quote = this.safeCurrencyCode (quoteId);
+        const symbol = base + '/' + quote;
+        market = this.safeMarket (symbol, market);
+        const time = this.safeInteger (order, 'time', this.milliseconds ());
+        let timestamp = time;
+        if (isTransaction) {
+            timestamp = this.parse8601 (time);
+        }
+        const canceled = this.safeValue (order, 'cancel', false);
+        let status = 'open';
+        if (canceled) {
+            status = 'canceled';
+        } else if (isTransaction) {
+            status = 'closed';
+        }
+        const parsedOrder = {
+            'id': this.safeString2 (order, 'id', 'order'),
+            'clientOrderId': undefined,
+            'info': order,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'status': status,
+            'symbol': symbol,
+            'type': undefined,
+            'timeInForce': undefined,
+            'postOnly': undefined,
+            'side': this.safeString (order, 'type'),
+            'price': this.safeNumber (order, 'price'),
+            'stopPrice': undefined,
+            'average': undefined,
+            'cost': undefined,
+            'amount': amount,
+            'filled': undefined,
+            'remaining': remaining,
+            'fee': {
+                'cost': this.safeNumber2 (order, 'fee', 'fee_amount'),
+                'currency': quote,
+                'rate': undefined,
+            },
+            'trades': undefined,
+        };
+        if (isTransaction) {
+            parsedOrder['trades'] = this.parseWsTrade (order, market);
+        }
+        return this.safeOrder (parsedOrder, market);
+    }
+
+    fromPrecision (amount, scale) {
+        if (amount === undefined) {
+            return undefined;
+        }
+        const precise = new Precise (amount);
+        precise.decimals = this.sum (precise.decimals, scale);
+        precise.reduce ();
+        return precise.toString ();
```

Comment:
something's wrong with the scale here (should be int and not float), and btw I don't think we should be using `toString` here, we should be using the appropriate method to convert numbers to strings. 
![image](https://user-images.githubusercontent.com/43336371/197579606-31ef7a6c-a691-4d9d-ac51-50a5d3a9dd50.png)

Reply:
Good catch, I figured it out it was an issue of the precision in `fetchCurrencies`